kafka-bin-scripts/README.adoc

```diff
@@ -46,74 +46,49 @@ ifdef::context[:parent-context: {context}]
 
 .Prerequisites
 ifndef::community[]
 * You have a Red Hat account.
 endif::[]
 * You have a running Kafka instance in {product}.
+* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^].
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
 * For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
 
-ifdef::qs[]
-[#description]
-Learn how to use Kafka bin scripts to interact with a Kafka instance in {product-long}.
-
-[#introduction]
-Welcome to the quick start for {product-long} with Kafka bin scripts. In this quick start, you'll learn how to use the Kafka bin scripts to produce and consume messages for your Kafka instances in {product}.
-endif::[]
-
-[id="proc-downloading-kafka-bin-scripts_{context}"]
-== Downloading the Kafka bin scripts
-
-The Kafka bin scripts are the binary scripts that are provided in the https://kafka.apache.org/downloads[Apache Kafka distribution^]. When you extract the Apache Kafka distribution, the `bin/` directory (or the `bin\windows\` directory if you're using Windows) of the distribution contains a set of shell scripts that enable you to interact with your Kafka instance. With the bin scripts, you can produce and consume messages, and perform various operations against the Kafka APIs to administer topics, consumer groups, and other resources.
-
-ifndef::community[]
-NOTE: The Kafka bin scripts are part of the open source community version of Apache Kafka. The bin scripts are not a part of {product} and are therefore not supported by Red Hat.
-endif::[]
-
-.Procedure
-. In a web browser, go to the Kafka https://kafka.apache.org/downloads[Download^] page and download the latest binary version of Apache Kafka.
-. Extract the downloaded archive and navigate to the `bin/` directory.
-+
---
-This example extracts the archive and then changes to the `bin/` directory.
-
-.Extracting the Kafka archive and navigating to the `bin/` directory
-[source]
-----
-$ tar -xzf kafka_2.13-2.7.0.tgz
-$ cd kafka_2.13-2.7.0/bin
-----
-
-The `bin/` directory contains the Kafka bin scripts.
---
-
-. Review the scripts in the `bin/` directory and verify that you have the `kafka-console-producer` and `kafka-console-consumer` scripts.
+You can check the version number of the `kafka-console-producer` script to verify that the expected Kafka version is downloaded.
 
-. Check the version number of the `kafka-console-producer` script to verify that the scripts were downloaded correctly.
-+
-.Verifying Kafka bin scripts
+.Verifying Kafka scripts
 [source]
 ----
 $ ./kafka-console-producer.sh --version
 2.7.0 (Commit:448719dc99a19793)
 ----
```

Comment:
@PaulRMellor I'm wondering if we could streamline the prereq a bit, something like this. I know it's not too common to put in a snippet in a prereq, but I've done it a couple of times and have enjoyed the payoff it brings. But up to you of course.

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
* You have a running Kafka instance in {product}.
* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^] and verified the installation.
+
.Verifying Kafka scripts
[source]
----
$ ./kafka-console-producer.sh --version
2.7.0 (Commit:448719dc99a19793)
----
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
```

Or possibly this:


```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
* You have a running Kafka instance in {product}.
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^] and verified the installation.
+
.Verifying Kafka scripts
[source]
----
$ ./kafka-console-producer.sh --version
2.7.0 (Commit:448719dc99a19793)
----
```

I'd suggest the same approach for the prereq in the Kafkacat bit too.

Reply:
I did think about that and I've used them before myself. Wasn't sure of the _quick start_ convention, so happy to go with option B here. :+1: